Patches/DefensiveMachineGunTurretPack/DMGT_CE_Patch_ThingDefs_Sentry_Turrets.xml

```diff
@@ -0,0 +1,237 @@
+						<minRange>6</minRange>
+						<ticksBetweenBurstShots>9</ticksBetweenBurstShots>
+						<burstShotCount>6</burstShotCount>
+						<soundCast>InfernoCannon_Fire</soundCast>
+						<soundCastTail>GunTail_Heavy</soundCastTail>
+						<muzzleFlashScale>16</muzzleFlashScale>
+						<recoilPattern>Mounted</recoilPattern>
+					</Properties>
+					<AmmoUser>
+						<magazineSize>48</magazineSize>
+						<reloadTime>7.8</reloadTime>
+						<ammoSet>AmmoSet_40x53mmGrenade</ammoSet>
+					</AmmoUser>
+					<FireModes>
+						<aiUseBurstMode>FALSE</aiUseBurstMode>
+						<aiAimMode>AimedShot</aiAimMode>
+						<aimedBurstShotCount>3</aimedBurstShotCount>
+						<noSnapshot>true</noSnapshot>
+						<noSingleShot>true</noSingleShot>
+					</FireModes>
+					<weaponTags>
+						<li>TurretGun</li>
+					</weaponTags>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Turret_Mk19MGLs"]/specialDisplayRadius</xpath>
+					<value>
+						<specialDisplayRadius>78</specialDisplayRadius>
+					</value>
+				</li>
+
+				<!-- ========== Work To Build ========== -->
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[
+					defName="Turret_M2HBs"
+				]/statBases/WorkToBuild</xpath>
+					<value>
+						<WorkToBuild>69500</WorkToBuild>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[
+					defName="Turret_M134MGs"
+				]/statBases/WorkToBuild</xpath>
+					<value>
+						<WorkToBuild>62500</WorkToBuild>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[
+					defName="Turret_Mk19MGLs"
+				]/statBases/WorkToBuild</xpath>
+					<value>
+						<WorkToBuild>66500</WorkToBuild>
+					</value>
+				</li>
+
+				<!-- ========== Common to all Sentry Turrets ========== -->
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[
+					defName="Turret_M2HBs" or
+					defName="Turret_M134MGs" or
+					defName="Turret_Mk19MGLs"
+				]/thingClass</xpath>
+					<value>
+						<thingClass>CombatExtended.Building_TurretGunCE</thingClass>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[
+					defName="Turret_M2HBs" or
+					defName="Turret_M134MGs" or
+					defName="Turret_Mk19MGLs"
+				]/researchPrerequisites</xpath>
+					<value>
+						<researchPrerequisites>
+							<li>CE_TurretHeavyWeapons</li>
+							<li>CE_HeavyTurret</li>
+						</researchPrerequisites>
+					</value>
+				</li>
+
+				<li Class="PatchOperationRemove">
+					<xpath>Defs/ThingDef[
+					defName="Turret_M2HBs" or
+					defName="Turret_M134MGs" or
+					defName="Turret_Mk19MGLs"
+				]/comps/li[@Class = "CompProperties_Refuelable"]</xpath>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[
+					defName="Turret_M2HBs" or
+					defName="Turret_M134MGs" or
+					defName="Turret_Mk19MGLs"
+				]/statBases</xpath>
+					<value>
+						<AimingAccuracy>1.0</AimingAccuracy>
```

Comment:
The vanilla mini-turret only has AimingAccuracy of 0.25 and ShoottingAccuracyTurret of 0.5. These values don't make them overly accurate, do they?

Reply:
Not really - actual in-game testing shows that the CE-patched Mk-19 turrets fire are actually _somewhat less_ accurate than the vanilla versions. Which works out just fine to how the turret is intended to be used.

(I should know this, since I am also the maintainer of the original mod)